### ISSUE TITLE
[History] Expanded Functionality of the History Feature

### ISSUE THREAD

**Issue by silent-arbiter** [Ryuasd/hQuestMaster#36]
In the chat history, show searching for traps / treasure / secret doors. Bonus points if the results of these can be referenced by clicking on them in the History panel. The quest text (lore, what the search revealed, etc.) could therefore be available by clicking on their entry.

**Comment by Ryuasd** [comment]
Yes those are good ideas, i try to implement those then i work again on the History.
